README.md

```diff
@@ -330,6 +330,26 @@ You can get the full list of supported receivers like so:
 env.receiver_options
 ```
 
+#### Delivering a payment method
+
+You can deliver a payment method to a third party using [Payment Method Distribution](https://docs.spreedly.com/guides/payment-method-distribution/). Once a receiver is set up and you have a payment method that you would like to share, you can use the following call:
+
+```ruby
+env.deliver_to_receiver(
+    receiver_token,       # String
```

Comment:
Instead of adding comments in the example for field types, it might be worth adding an example usage. 

```rb
env.deliver_to_receiver(
  "RECEIVER TOKEN HERE",
  "PAYMENT METHOD TOKEN HERE",
  headers: { "Content-Type": "application/json" },
  url: "https://spreedly-echo.herokuapp.com",
  body: { card_number: "{{credit_card_number}}" }.to_json
)
```

Reply:
I added the typing only because the headers field deviates from the other fields but I think that your example looks cleaner and would be easier to copy/paste so I will switch over to that :+1: